apps/guardian-ui/src/components/setup/screens/setConfiguration/FederationSettingsForm.tsx

```diff
@@ -24,6 +24,8 @@ export const FederationSettingsForm: React.FC<FederationSettingsFormProps> = ({
 }) => {
   const { t } = useTranslation();
 
+  const bftNumbers = [4, 7, 10, 13, 16, 19];
```

Comment:
nit: this should probably go into a `util/constants` file... 

Also should add `as const` at the end to make it readonly